app/assets/stylesheets/src/layout/attendace.css.scss

```diff
@@ -0,0 +1,44 @@
+@import "../base/variables.css";
+
+.table__filter{
+    
```

Comment:
Is there any reason to have these blank lines here? 🤔

Reply:
Done